.env.example

```
@@ -0,0 +1,41 @@
+# Stellar Goal Vault Environment Variables Example
+# Copy this file to .env and fill in the actual values
+
+# Server Configuration
+PORT=3000
+NODE_ENV=development
+
+# Stellar Network Configuration
+NETWORK_PASSPHRASE=Test SDF Network ; September 2015
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)